cli/tests/testdata/npm/cjs_with_deps/main_info.out

```diff
@@ -4,19 +4,19 @@ dependencies: 14 unique
 size: [WILDCARD]
 
 file:///[WILDCARD]/npm/cjs_with_deps/main.js ([WILDCARD])
-├─┬ npm:chalk@4.1.2 ([WILDCARD])
-│ ├─┬ npm:ansi-styles@4.3.0 ([WILDCARD])
-│ │ └─┬ npm:color-convert@2.0.1 ([WILDCARD])
-│ │   └── npm:color-name@1.1.4 ([WILDCARD])
-│ └─┬ npm:supports-color@7.2.0 ([WILDCARD])
-│   └── npm:has-flag@4.0.0 ([WILDCARD])
-└─┬ npm:chai@4.3.6 ([WILDCARD])
-  ├── npm:assertion-error@1.1.0 ([WILDCARD])
-  ├── npm:check-error@1.0.2 ([WILDCARD])
-  ├─┬ npm:deep-eql@3.0.1 ([WILDCARD])
-  │ └── npm:type-detect@4.0.8 ([WILDCARD])
-  ├── npm:get-func-name@2.0.0 ([WILDCARD])
-  ├─┬ npm:loupe@2.3.4 ([WILDCARD])
-  │ └── npm:get-func-name@2.0.0 ([WILDCARD])
-  ├── npm:pathval@1.1.1 ([WILDCARD])
-  └── npm:type-detect@4.0.8 ([WILDCARD])
+├─┬ npm:/chalk@4.1.2 ([WILDCARD])
+│ ├─┬ npm:/ansi-styles@4.3.0 ([WILDCARD])
+│ │ └─┬ npm:/color-convert@2.0.1 ([WILDCARD])
+│ │   └── npm:/color-name@1.1.4 ([WILDCARD])
+│ └─┬ npm:/supports-color@7.2.0 ([WILDCARD])
+│   └── npm:/has-flag@4.0.0 ([WILDCARD])
+└─┬ npm:/chai@4.3.6 ([WILDCARD])
+  ├── npm:/assertion-error@1.1.0 ([WILDCARD])
+  ├── npm:/check-error@1.0.2 ([WILDCARD])
```

Comment:
Why this change?

Reply:
The resolved specifier for an npm module now includes the `/`, so the top level specifiers displaying here were also having the foward slash:

```
file:///V:/deno/cli/tests/testdata/npm/cjs_with_deps/main.js (325B)
├─┬ npm:/chalk@4.1.2 (34.23KB)
│ ├─┬ npm:ansi-styles@4.3.0 (16.58KB)
│ │ └─┬ npm:color-convert@2.0.1 (26.55KB)
│ │   └── npm:color-name@1.1.4 (6.54KB)
│ └─┬ npm:supports-color@7.2.0 (6.87KB)
│   └── npm:has-flag@4.0.0 (4.32KB)
└─┬ npm:/chai@4.3.6 (732.97KB)
  ├── npm:assertion-error@1.1.0 (5.51KB)
  ...
```

To make it consistent I added the forward slash to everything. I think it's more correct?

Reply:
Yeah, I think it makes sense, but does that mean that we will require `npm:/chai` in the import statement as well? I guess not

Reply:
It won't be required